.contentful.json.sample

```diff
@@ -8,4 +8,4 @@
     "spaceId": "...",
     "accessToken": "..."
   }
-}
+}
```

Comment:
```suggestion
}

```

The most nittest of nits, convention is to end in a newline because unix said so 50 years ago. *Almost* nothing breaks anymore because of this but git has carried the tradition of putting that warning indicator on when you don't end a file in a newline